wagtail/locks.py

```diff
@@ -151,22 +149,46 @@ def get_message(self, user):
         if self.for_user(user):
             if len(self.object.current_workflow_state.all_tasks_with_status()) == 1:
                 # If only one task in workflow, show simple message
-                workflow_info = _("This page is currently awaiting moderation.")
+                if self.is_page:
+                    workflow_info = _("This page is currently awaiting moderation.")
+                else:
+                    workflow_info = capfirst(
```

Comment:
> Is it in case the translation has model name first and the model name isn't capitalized?

Yep

> Does capfirst know that some languages don't have capitalization?

It uses Python's `upper()` so it should handle it gracefully. If the language doesn't have capitalisation then it doesn't matter, no?

Reply:
Yep, this was just a check. As mentioned, these were more notes to double check later.